src/backend/cuda/include/yaksuri_cudai.h

```diff
@@ -16,6 +16,9 @@
 #define CUDA_P2P_CLIQUES  (3)
 
 #define YAKSURI_CUDAI_THREAD_BLOCK_SIZE  (256)
```

Comment:
The max x or y block size is 1024. Is this lower than that because you want to allow the pack/unpack to share SMs with user kernels? Smaller block size will imply more context switches among the thread blocks, thus, if sharing is not a concern maybe this should be increased?

Reply:
Right.  There's a separate issue for tuning the number of threads and block sizes (pmodels/yaksa#30).

Reply:
Thanks! I missed that